build/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <script src="index.js"></script>
+    <link rel="stylesheet" type="text/css" href="index.css">
+  </head>
+  <body>
+    <canvas id="root" style="outline:1px solid red;"></canvas>
```

Comment:
`outline` instead of `border` prevents the `border` from increasing the `canvas`'s width which helps when checking if the snake leaves the game screen.